### PR TITLE
Update README, add LICENSE and notice.md files

### PR DESCRIPTION
**Pull request by okeefm**
This PR:
* Adds a `LICENSE` file (Apache 2.0)
* Adds a `notice.md` file, based on the CMS/HHS 2021 Data Rights Legend (modified for open source use)
* Updates the `README.md` to detail all the various options available in the app, how to run it locally, and a link to the hosted version

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (lmd59) [inline LICENSE:189]:
```suggestion
   Copyright 2021 The MITRE Corporation
```

File: README.md [hunk 2]
```diff
@@ -1 +1,73 @@
 # fqm-execution-demo
+
+This is a React app that demonstrates the capabilities of the [fqm-execution](https://github.com/projecttacoma/fqm-execution) library in calculating FHIR-based electronic Clinical Quality Measures (eCQMs).
+
+## Hosted Version
+
+This app is hosted on GitHub Pages, at [https://projecttacoma.github.io/fqm-execution-demo/](https://projecttacoma.github.io/fqm-execution-demo/), for use without standing up a local development environment.
```
Comment (lmd59) [inline README.md]:
```suggestion
This app is hosted on GitHub Pages, at [https://projecttacoma.github.io/fqm-execution-demo/](https://projecttacoma.github.io/fqm-execution-demo/), for demonstration use without standing up a local development environment.
```
Just to make it extra clear that it's a proof of concept

File: README.md [hunk 3]
```diff
@@ -1 +1,73 @@
...
+* By selecting a measure bundle from the `Select from Connectathon Repository` dropdown list, which is populated with the measure bundles from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository.
+* By selecting a measure bundle from the `Select from eCQM Measure Content Repository` dropdown, which is populated from the [cqframework/ecqm-content-r4](https://github.com/cqframework/ecqm-content-r4) repository.
+
+### Patient Bundle
+
+In order to run calculation, the demo app requires a `Bundle` resource that includes the `Patient` resource, and all other required clinical resources (`Procedure`s, `Encounter`s, etc.). Users can provide patient bundles in four different ways:
+
+* By uploading a patient bundle from their local computer, by clicking on the box below `Upload From File System` or dragging a bundle from their file browser to that box.
+* By selecting a patient bundle from the `Select from Connectathon Repository` dropdown list, which is populated from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository if a matching measure is selected in the `Measure Bundle` field.
```
Comment (lmd59) [inline README.md]:
```suggestion
* By selecting a patient bundle from the `Select From Connectathon Repository` dropdown list, which is populated from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository if a matching measure is selected in the `Measure Bundle` field.
```
to match application

File: README.md [hunk 4]
```diff
@@ -1 +1,73 @@
...
+This app is hosted on GitHub Pages, at [https://projecttacoma.github.io/fqm-execution-demo/](https://projecttacoma.github.io/fqm-execution-demo/), for use without standing up a local development environment.
+
+## Options/fields
+
+### Measure Bundle
+
+In order to run calculation, the demo app requires a `Bundle` resource that includes the `Measure` resource, all required `Library` resources, and all required `ValueSet` resources. Users can provide measure bundles in three different ways:
+
+* By uploading a measure bundle from their local computer, by clicking on the box below `Upload From File System` or dragging a bundle from their file browser to that box.
+* By selecting a measure bundle from the `Select from Connectathon Repository` dropdown list, which is populated with the measure bundles from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository.
```
Comment (lmd59) [inline README.md]:
```suggestion
* By selecting a measure bundle from the `Select From Connectathon Repository` dropdown list, which is populated with the measure bundles from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository.
```

File: README.md [hunk 5]
```diff
@@ -1 +1,73 @@
...
+* By selecting a measure bundle from the `Select from eCQM Measure Content Repository` dropdown, which is populated from the [cqframework/ecqm-content-r4](https://github.com/cqframework/ecqm-content-r4) repository.
```
Comment (lmd59) [inline README.md]:
```suggestion
* By selecting a measure bundle from the `Select From eCQM Measure Content Repository` dropdown, which is populated from the [cqframework/ecqm-content-r4](https://github.com/cqframework/ecqm-content-r4) repository.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,73 @@
...
+
+### Patient Bundle
+
+In order to run calculation, the demo app requires a `Bundle` resource that includes the `Patient` resource, and all other required clinical resources (`Procedure`s, `Encounter`s, etc.). Users can provide patient bundles in four different ways:
+
+* By uploading a patient bundle from their local computer, by clicking on the box below `Upload From File System` or dragging a bundle from their file browser to that box.
+* By selecting a patient bundle from the `Select from Connectathon Repository` dropdown list, which is populated from the [DBCG/connectathon](https://github.com/dbcg/connectathon) repository if a matching measure is selected in the `Measure Bundle` field.
+* By selecting a patient bundle from the `Select from eCQM Measure Content Repository` dropdown, which is populated from the [cqframework/ecqm-content-r4](https://github.com/cqframework/ecqm-content-r4) repository if a matching measure is selected in the `Measure Bundle` field.
```
Comment (lmd59) [inline README.md]:
```suggestion
* By selecting a patient bundle from the `Select From eCQM Measure Content Repository` dropdown, which is populated from the [cqframework/ecqm-content-r4](https://github.com/cqframework/ecqm-content-r4) repository if a matching measure is selected in the `Measure Bundle` field.
```

File: README.md [hunk 7]
```diff
@@ -1 +1,73 @@
...
+* By selecting a patient bundle from the `Select from FHIR Patient Generator Repository` dropdown list, which is populated from the [projecttacoma/fhir-patient-generator] repository if a matching measure is selected in the `Measure Bundle` field.
```
Comment (lmd59) [inline README.md]:
```suggestion
* By selecting a patient bundle from the `Select From FHIR Patient Generator Repository` dropdown list, which is populated from the [projecttacoma/fhir-patient-generator] repository if a matching measure is selected in the `Measure Bundle` field.
```

Comment (lmd59) [inline README.md]: Maybe switch the order of these last two to match the application's order

File: LICENSE (hunk 1 above)
Comment (okeefm) [reply LICENSE:189]: I think this one is technically an "example" of how to use the Apache License? The actual one that has the copyright is down in our `README`

File: README.md (hunk 2 above)
Comment (okeefm) [reply README.md]: Makes sense to me!

**Review by lmd59 — APPROVED** [review]

**Pull request merged**